dataverse-webapp/src/main/webapp/roles-edit.xhtml

```diff
@@ -62,7 +62,8 @@
                         </div>
                     </div>
                 </div>
-                <p:watermark value="#{bundle['dataverse.permissions.roles.watermark.required']}" for="roleName roleAlias" id="requiredWatermark" />
+                <p:watermark value="#{bundle['dataverse.permissions.roles.watermark.required']}" for="roleName" />
+                <p:watermark value="#{bundle['dataverse.permissions.roles.watermark.required']}" for="roleAlias" />
```

Comment:
It would be easier for future changes if we put watermarks next to roleName and roleAlias inputs respectively.

Reply:
After consulting with Wojtek we can get rid of these watermarks.